lib/kino/js/live.ex

```diff
@@ -322,16 +327,25 @@ defmodule Kino.JS.Live do
 
   The given `init_arg` is passed to the `init/2` callback when
   the underlying kino process is started.
+
+  ## Options
+
+    * `:export` - a function called to export the given kino to Markdown.
+      This works the same as `Kino.JS.new/3`, except the function
+      receives `t:Kino.JS.Live.Context.t/0` as an argument
```

Comment:
@josevalim I made this an option instead of a callback, for two reasons:

1. It better mirrors `Kino.JS`, so should be more intuitive. We also have the option on `Kino.Table`, this way they are all consistent.

2. In some cases the original data may not be passed to the server directly (see [Kino.DataTable.new/2](https://github.com/livebook-dev/kino/blob/8dac6e2b87455f9126b8860c6c3763dadc9d136c/lib/kino/data_table.ex#L69)). Passing it to the state for the purpose of export is not different from closure, but if it's large data then it may be better to just call `inspect/1` right away and close over that.

Reply:
Also, I'm not sure if should actually be passing the state to that function. If we do that, then people may make the export reflect the state (e.g. page when paginating the table). It is undesired, because (a) we wouldn't know that we should re-save when the page changes, so it would end up pretty arbitrary (b) it wouldn't reflect the actual value, that is, if the cell returns `df` and we suddenly persist 10th page, it's weird. Thoughts?